jwt.js

```diff
@@ -192,7 +192,7 @@ function fastifyJwt (fastify, options, next) {
         let scheme = parts[0]
         token = parts[1]
 
-        if (!/^Bearer$/i.test(scheme)) {
+        if (scheme !== 'Bearer') {
```

Comment:
Note that the regexp matches `'bearer'` as well. Is this an expected change? I think http headers are meant to be case insensitive.

cc @jsumners

Reply:
uh, i miss the `i` flag.

Reply:
Typically it’s only the header names that are case insensitive. Whether or not the values are insensitive is up to the server implementation.